src/framework/core/js/Fluid.js

```diff
@@ -826,6 +826,28 @@ var fluid = fluid || fluid_3_0_0;
         return isFinite(string) && ((string % 1) === 0) ? Number(string) : NaN;
     };
 
+    /**
+     * Derived from AGK's stack overflow answer ( http://stackoverflow.com/a/12830454 )
+     *
+     * Rounds the supplied number to at most the number of decimal places indicated by the scale, omiting any trailing 0s.
+     * If the scale is invalid (i.e falsey, not a number, negative value), it is treated as 0.
+     * If the scale is a floating point number, it is rounded to an integer.
+     *
+     * @param {number} num - the number to be rounded
```

Comment:
We tend to use capital letters for these types - "Number", as per the ECMAScript standard - https://www.ecma-international.org/ecma-262/5.1/#sec-4.3.19 - even though, of course, typeof tests have to use the form with a lower case letter.